crates/bevy_ui/src/entity.rs

```diff
@@ -136,22 +134,50 @@ impl Default for ButtonBundle {
         }
     }
 }
-#[derive(Component, Clone)]
-pub struct CameraUi {
-    pub is_enabled: bool,
-}
 
-impl Default for CameraUi {
+/// Configuration for cameras related to UI.
+///
+/// When a [`Camera`] doesn't have the [`CameraUiConfig`] component,
+/// it will display the UI by default.
+///
+/// [`Camera`]: bevy_render::camera::Camera
+#[derive(Component, Reflect, Clone, Debug)]
+pub struct CameraUiConfig {
+    /// Whether to output UI to this camera view.
+    ///
+    /// When a `Camera` doesn't have the [`CameraUiConfig`] component,
+    /// it will display the UI by default.
+    pub show_ui: bool,
+    /// Scale of the UI.
+    pub scale: f32,
+    /// Position of the camera compared to the UI.
+    pub position: Vec2,
+}
+impl Default for CameraUiConfig {
     fn default() -> Self {
-        Self { is_enabled: true }
+        Self {
+            show_ui: false,
```

Comment:
wouldn't the better default be true here ?

Reply:
I think it might, you are right. Ahah, just reverting the change from my previous PR :)